README.md

```diff
@@ -22,6 +23,122 @@ git add -A
 
 This section provides instructions on how you can enable CI for your repository.
 
+### Github Actions
+
+These are instructions when using [Github
+Actions](https://docs.github.com/en/free-pro-team@latest/actions).
+
+The following configuration shows how to:
+
+* Upload nuget packages to a NuGet feed for tags, and commits on `main` branch.
+
+  This requires adding a secret named `NUGET_API_KEY` to your
+  repository on GitHub.
+
+  Add a `.github/workflows/ci.yml` to your repository:
+
+  ``` yml
+  name: Build, test and publish NuGet packages
+
+  on:
+  push:
+      branches: [ main ]
+      tags:
+      - '*'
+
+  jobs:
+  build:
+      name: Build, test and publish NuGet package
+
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+          dotnet-version: '5.0.x'
+      - name: Restore dependencies
+      run: dotnet restore
+      - name: Build
+      run: dotnet build --no-restore
+      - name: Test
+      run: dotnet test  --verbosity normal
+      - name: Publish packages
+      run: dotnet msbuild -t:PushPackages
+      env:
+          NuGetApiKey: ${{ secrets.NUGET_API_KEY }}
+      - name: Save Artifacts
+      uses: actions/upload-artifact@v2
+      with:
+          name: nuget-package
+          path: artifacts/packages/Release/*.nupkg
+  ```
+
+* Create draft releases on GitHub when pushing a tag
+
+  Add this `.github/workflows/release.yml` to your repository:
+
+  ``` yml
+   name: Create draft release on tags
+
+   on:
+     push:
+       tags:
+         - '*'
+
+   jobs:
+     build:
+       name: Build and test and draft a release
+
+       runs-on: ubuntu-latest
+
+       steps:
+         - uses: actions/checkout@v2
+         - name: Setup .NET
+           uses: actions/setup-dotnet@v1
+           with:
+             dotnet-version: '5.0.x'
+         - name: Restore dependencies
+           run: dotnet restore
+         - name: Build
+           run: dotnet build --no-restore
+         - name: Test
+           run: dotnet test  --verbosity normal
+         - name: Build packages
+           id: package
+           run: |
+             set -euo pipefail
+             dotnet msbuild -t:"Restore;Pack"
```

Comment:
I think we can remove this `dotnet msbuild` invocation? The build step before already created packages.

Reply:
Maybe name the step 'Find package'